README.md

```diff
@@ -2,179 +2,128 @@
+`
 
-note: some gtp2ogsarguments have default so they are enabled even if you don't
-specify them.
+### For Linux
 
-### Example of use
+General use case is something like this:
 
-We are using the AI PhoenixGo on linux in this example,
-but it works the same way on other platforms and AI:
+`
+node ~/gtp2ogs-node/node_modules/gtp2ogs/gtp2ogs.js C:\Users\yourusername\gtp2ogs-node\node_modules\gtp2ogs\gtp2ogs.js --username <yourbotusername> --apikey <apikey> <gtp2ogsargument1> <gtp2ogsargument2> -- /path/to/your/ai/executable <botargument1> <botargument2>
+`
 
-```Shell
-sudo node /usr/lib/node_modules/gtp2ogs/gtp2ogs.js --beta --apikey 5rg46ze84f68e4g6v4e4g68es4g8 --username testbot --debug --startupbuffer 2 --bans 454,someplayer,489,512 --noclock --unrankedonly --maxconnectedgames 10 --maxconnectedgamesperuser 1 --minrank 5d --noautohandicapranked --maxhandicap 0 --fakerank 8d --boardsizes 19 --komis 7.5 --speeds live --maxmaintimelive 600 --maxperiodsliveranked 5 --maxperiodsliveunranked 10 --maxperiodtimeliveranked 60 --maxperiodtimeliveunranked 180 -- /home/amd2019/PhoenixGo/bazel-bin/mcts/mcts_main --gtp --config_path=/home/amd2019/PhoenixGo/etc/mcts_1gpu_beta.conf --v=1
-```
+For example:
 
-### Submit Move Button
+`
+node ~/gtp2ogs-node/node_modules/gtp2ogs/gtp2ogs.js --beta --apikey 5rg46ze84f68e4g6v4e4g68es4g8 --username somebot --debug --startupbuffer 2 --bans 454,someplayer,489,512 --noclock --unrankedonly --maxconnectedgames 10 --maxconnectedgamesperuser 1 --minrank 5d --noautohandicapranked --maxhandicap 0 --boardsizes 19 --komis 7.5 --speeds live --maxmaintimelive 600 --maxperiodsliveranked 5 --maxperiodsliveunranked 10 --maxperiodtimeliveranked 60 --maxperiodtimeliveunranked 180 -- /home/amd2020/PhoenixGo/bazel-bin/mcts/mcts_main --gtp --config_path=/home/amd2020/PhoenixGo/etc/mcts_1gpu_beta.conf --logtostderr --v=1
+`
 
-To avoid accidental misclicks while spectating a game from
-your bot account,
+## Submit Move Button
 
-see: [Submit-Move Button](https://github.com/wonderingabout/gtp2ogs-tutorial#important-submit-move-button)
+To avoid accidental misclicks while spectating a game from your bot account, see
+ [Submit-Move Button](https://github.com/wonderingabout/gtp2ogs-tutorial#important-submit-move-button).
 
-### Show winrate and variations ingame
+## Show winrate and variations ingame
```

Comment:
why is this a section here at all, rather than somewhere in options?

Reply:
because it's an important add to gtp2ogs, so i think it's worth giving it extra visibility (just a link to the option details)